logger/clickhouse.cc

```diff
@@ -195,3 +265,19 @@ tfw_column_factory(ch::Type::Code code)
 		throw std::runtime_error("Column factory: incorrect code");
 	}
 }
+
+template void
+TfwClickhouse::append_int<ch::ColumnUInt8, unsigned char>(TfwBinLogFields,
+							  unsigned char) noexcept;
+template void
+TfwClickhouse::append_int<ch::ColumnUInt16, uint16_t>(TfwBinLogFields,
+						      uint16_t) noexcept;
+template void
+TfwClickhouse::append_int<ch::ColumnUInt32, uint32_t>(TfwBinLogFields,
+						      uint32_t) noexcept;
+template void
+TfwClickhouse::append_int<ch::ColumnUInt64, uint64_t>(TfwBinLogFields,
+						      uint64_t) noexcept;
+template void
+TfwClickhouse::append_int<ch::ColumnIPv6, in6_addr>(TfwBinLogFields,
+						    in6_addr) noexcept;
```

Comment:
It's still better to get rid of douple specification of the type in the template since `read_int()` also propagates the double typing to `append_int()`